### PR TITLE
Add support for helmfile components without using EKS

### PR DESCRIPTION
**Pull request by stoned**
## what
Allow Helmfile component without using EKS.

## why
Because atmos could be used without using EKS.

### CODE REVIEW

File: internal/exec/helmfile.go [hunk 1]
```diff
@@ -51,6 +51,12 @@ func ExecuteHelmfile(cmd *cobra.Command, args []string) error {
 			"by 'metadata.type: abstract' attribute", path.Join(info.ComponentFolderPrefix, info.Component))
 	}
 
+	supportEKS := false
+	envVarsEKS := []string{}
+	if len(c.Config.Components.Helmfile.HelmAwsProfilePattern) > 0 {
```
[the remark this answers is not in the document]
Comment (stoned) [reply internal/exec/helmfile.go:56]: Sure !

[the remark this answers is not in the document]
Comment (stoned) [reply internal/exec/helmfile.go]: I pushed a variant. @aknysh let me know what you think about it.

**Comment by aknysh** [comment]
@stoned  thank for the update, please resolve the conflicts

**Comment by stoned** [comment]
> @stoned thank for the update, please resolve the conflicts

@aknysh I  pushed an update in which I also added support for a `ATMOS_COMPONENTS_HELMFILE_USE_EKS` environment variable.

**Comment by aknysh** [comment]
> > @stoned thank for the update, please resolve the conflicts
> 
> @aknysh I pushed an update in which I also added support for a `ATMOS_COMPONENTS_HELMFILE_USE_EKS` environment variable.

@stoned thanks again, I'll review it

**Pull request merged**